.gitignore

```diff
@@ -22,5 +22,5 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-dist/
+public/
```

Comment:
if you public is in the gitignore, how are you going to share the index.html page and also the css style pages?